lib/backup/logger/logfile.rb

```diff
@@ -102,8 +107,8 @@ def setup_logfile
           path = File.join(Backup::Config.root_path, path)
         end
         FileUtils.mkdir_p(path)
-        log_file = @options.log_file || 'backup.log'
-        path = File.join(path, log_file)
+        log_filename = @options.log_filename.strip.empty? ? 'backup.log' : @options.log_filename
```

Comment:
Hey! Not the author but it seems as though you are right. options.log_filename will already be stripped by the time it makes it to this point! I could branch from here and make the change to get this merged if need be!